server/src/dbt_execution/Dbt.ts

```diff
@@ -10,13 +10,25 @@ export enum DbtMode {
 }
 
 export abstract class Dbt {
-  constructor(private connection: _Connection, protected progressReporter: ProgressReporter) {}
+  dbtReady: boolean;
```

Comment:
Can we remove `abstract isReady(): Promise<void>;` method and use `isReady(): boolean;` instead of this variable?

Reply:
Also if I understand it correctly we should delete readyDeferred from `DbtCli`.

Reply:
Method `abstract isReady(): Promise<void>; is unused`. Removed.